### ISSUE TITLE
Generating Datasets

### ISSUE THREAD

**Issue by gaps013** [ndb796/Pytorch-Adversarial-Training-CIFAR#3]
Hi,

Could you please tell me how you generated the datasets?

I have been trying to generate the non-robust dataset, however, I am unable to get good test accuracy on that.

Thank you

**Comment by saigontrade88** [comment]
The author has provided a reference paper. You might take a look at the [paper](https://arxiv.org/pdf/1905.02175.pdf). He might get the data from the paper's online page.
